### ISSUE TITLE
Technical leadership needs a mention

### ISSUE THREAD

**Issue by hadleybeeman** [w3c/AB-public#221]
One of the W3C's biggest contributions to the web community is technical leadership. The TAG thinks this needs an explicit mention in the W3C vision. 

Perhaps in section [4. Vision for W3C](https://www.w3.org/TR/w3c-vision/)?

**Comment by michaelchampion** [comment]
https://www.w3.org/TR/w3c-vision/#vision-org already begins with 

> W3C leads the community in defining a World Wide Web that puts users first, by developing principles-based technical standards and guidelines.

The TAG "leads" by contributing to those principles and guidelines.  What more explicit mention does the TAG want to see in the Vision?

**Comment by martinthomson** [comment]
Is this a duplicate of #215?

**Comment by cwilso** [comment]
I don't think this is a duplicate of #215; that issue is more about explicitly charting a path for the Web - as you (@martinthomson) put it, the "what".  This is mentioning the role - the "how". 

I don't think we need to overstate this.  We do (as @michaelchampion said) lead off the "Vision for W3C" section with a sentence that includes both "leadership" and "technical", so I think this already has an explicit mention.

**Comment by cwilso** [comment]
The AB concluded (based on the above) to close this.
